tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel\Functional;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests that the Person content type renders correctly.
+ */
+class ContentPersonRenderTest extends WhitelabelBrowserTestBase {
+
+  use MediaTypeCreationTrait;
+  use TestFileCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
+    'oe_whitelabel_starter_person',
+  ];
+
+  /**
+   * Tests the person page.
+   */
+  public function testPersonPage(): void {
+    $node = $this->createExamplePersonWithRequiredFieldsOnly();
+    $this->drupalGet($node->toUrl());
+    /** @var \Symfony\Component\BrowserKit\AbstractBrowser $client */
+    $client = $this->getSession()->getDriver()->getClient();
+    $crawler = $client->getCrawler();
+
+    // Assert required fields are present.
+    $content_banner = $crawler->filter('.bcl-content-banner');
+    $this->assertCount(1, $content_banner);
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert optional fields are not present.
+    // Assert no image is present.
+    $this->assertCount(0, $content_banner->filter('img'));
+
+    // Assert short description is not present.
+    $this->assertCount(0, $content_banner->filter('p'));
+
+    // Assert no social media links are present.
+    $this->assertCount(0, $content_banner->filter('a.standalone'));
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
+
+    // Assert additional information is not present.
+    $this->assertStringNotContainsString(
+      'Additional information',
+      $content->text()
+    );
+
+    // Assert documents are not present.
+    $this->assertStringNotContainsString(
+      'Related documents',
+      $content->text()
+    );
```

Comment:
I think we should avoid these super-specific selectors for _negative_ assertions.
Otherwise, if any of the classes above is removed, the assertions will no longer fail if the text is present.
Instead, I think we can just assert that the labels are nowhere to be found in the page.

```suggestion
    // Assert additional information is not present.
    $this->assertStringNotContainsString(
      'Additional information',
      $crawler->text()
    );

    // Assert documents are not present.
    $this->assertStringNotContainsString(
      'Related documents',
      $crawler->text()
    );
```